packages/shared/src/utils/contract/contract-types.ts

```diff
@@ -0,0 +1,80 @@
+import { ContractUpdate } from '@deriv/api-types';
+
+export type TGetFinalPrice = {
+    sell_price: number;
+    bid_price: number;
+};
+
+export type TIsValidToSell = TIsEnded & {
+    is_valid_to_sell: 0 | 1;
+};
+
+export type TGetTotalProfit = {
+    bid_price: number;
+    buy_price: number;
+};
+
+export type TGetDisplayStatus = TGetTotalProfit & {
+    status: 'open' | 'sold' | Omit<string, 'open' | 'sold'>;
+};
+
+export type TTickStream = {
+    epoch?: number;
+    tick?: null | number;
+    tick_display_value?: null | string;
+};
+
+export type TIsEnded = {
+    sell_price?: number;
+    bid_price?: number;
+    is_valid_to_sell?: 0 | 1;
+    status?: 'open' | 'sold' | Omit<string, 'open' | 'sold'>;
+    is_expired?: 0 | 1;
+    is_settleable?: 0 | 1;
+};
+
+export type TContractInfo = {
+    tick_stream?: TTickStream[];
+    cancellation?: { ask_price: number };
+    status?: 'open' | 'sold' | Omit<string, 'open' | 'sold'>;
```

Comment:
I believe we can make a separate reusable type

Reply:
As I understand, we can just use 'string' here.
or if we are sure what kind of status could be we can stay just with 'open' | 'sold'